irohad/consensus/yac/cluster_order.hpp

```diff
@@ -49,13 +55,20 @@ namespace iroha {
          */
         bool hasNext();
 
-        std::vector<model::Peer> getPeers() { return order_; };
+        std::vector<model::Peer> getPeers() {
+          return order_;
+        };
 
-        auto getNumberOfPeers() { return order_.size(); }
+        auto getNumberOfPeers() {
+          return order_.size();
+        }
 
         virtual ~ClusterOrdering() = default;
 
        private:
+        explicit ClusterOrdering(std::vector<model::Peer> order);
+        ClusterOrdering() {}
```

Comment:
what is the reason for presence of empty ctor here?

Reply:
To make sure that creation of this object is only possible with create method

Reply:
public:
ClusterOrdering() = delete;

Reply:
https://stackoverflow.com/questions/13654927/why-explicitly-delete-the-constructor